gapis/server/export_replay.go

```diff
@@ -83,9 +92,9 @@ func exportReplay(ctx context.Context, c *path.Capture, d *path.Device, out stri
 						req.Attachment,        // api.FramebufferAttachment
 						fbInfo.Index,          // uint32
 						req.Settings.DrawMode, // service.DrawMode
-						true,                  // disableReplayOptimization bool
-						false,                 // displayToSurface bool
-						nil,                   // hints *service.UsageHints
+						true,  // disableReplayOptimization bool
```

Comment:
Format looks broken here? (comments should be tabbed to align the ones of previous lines)